src/components/Banners/GithubUsernameBanner.tsx

```diff
@@ -0,0 +1,47 @@
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
+import { uiColors } from "@leafygreen-ui/palette";
+import { useLocation } from "react-router-dom";
+import { paths } from "constants/routes";
+import { size } from "constants/tokens";
+import { GetUserSettingsQuery } from "gql/generated/types";
+import { GET_USER_SETTINGS } from "gql/queries";
+
+const { green } = uiColors;
+export const GithubUsernameBanner = () => {
+  const { pathname } = useLocation();
+  const isPatchesPage = pathname.startsWith(paths.user);
```

Comment:
Does this make sense to live in the `PatchesPage` component. That way it will always be rendered on a Patches Page
https://github.com/evergreen-ci/spruce/blob/f7a503ba9a4f553655dbfca46fbba6cf5e500cfc/src/components/PatchesPage/index.tsx

Reply:
Otherwise react-router exports a matchPath function. https://github.com/evergreen-ci/spruce/blob/e036af8357365d7be7cc6513b4a6e49af53b626b/src/hooks/useLegacyUIURL.ts